pkg/cmd/roachtest/backup.go

```diff
@@ -262,4 +275,163 @@ func registerBackup(r *testRegistry) {
 			m.Wait()
 		},
 	})
+
+	r.Add(testSpec{
+		Name:       "backupKMS",
```

Comment:
This test feels like it's testing RESTORE just as much as BACKUP (ensuring that we can restore with either KMS). As we add more, I also wonder if it's worthwhile to group these tests into something like "backup/{2TB,KMS}". Do you have thoughts here?